src/main/resources/tree.json

```diff
@@ -1,28 +1,33 @@
 {
   "key": "START",
```

Comment:
Could you please, draw a visualization of this tree? Even physical on paper will be enough.

Reply:
[chatbot-scheme.pdf](https://github.com/user-attachments/files/16522953/chatbot-scheme.pdf)